### DISCUSSION TITLE
Power Automate Parameters - Formatting Error

### DISCUSSION THREAD

**Discussion by ChadAlt** [rappen/FetchXMLBuilder#433] [General]
Hi Jonas, 

I figured I'd take advantage of the new community here. 👋

The Power Automate Parameters option was a genius addition to the tool! I love it. While using it, I found a specific example of inaccurate formatting being provided. When querying the connection entity, I am filtering my flow using the record1idobjecttypecode field which specifies the entity code. This value should be referenced without quote marks, but the provided Power Automate Parameters include quotes as shown below. 

(record1idobjecttypecode eq '8')

![image](https://user-images.githubusercontent.com/60116323/101820777-af240b80-3aec-11eb-96aa-157cfb1db287.png)

Perhaps this is happening in other entity references, too, but I haven't tried. Hopefully you find this feedback helpful. 

Love the tool, and the community appreciates the work you've done!

**Comment by rappen** [comment]
Thanks Chad!
This seems like a bug, I'll make sure it gets fixed :)
You may follow issue #434 to track progress...
